.github/workflows/ci_cron_weekly.yml

```diff
@@ -48,3 +48,23 @@ jobs:
         python -m pip install tox
     - name: Test with tox
       run: tox -e py310-test-devdeps
+
+  ci_roman_cron_tests:
```

Comment:
I wonder if we should just merge this into the job above. What is the point of having two variants of dev test for weekly cron when we already run a bulk of dev test in PRs?